helmfile.d/10-services.yaml

```diff
@@ -139,15 +139,37 @@ releases:
       - name: ingress.tls[0].hosts
         values: ["{{ .Values.server_name }}"]
 
+  - name: radar-kafka
+    chart: radar/radar-kafka
+    version: {{ .Values.radar_kafka_stack._chart_version }}
+    installed: {{ .Values.radar_kafka_stack._install }}
+    timeout: {{ add .Values.base_timeout .Values.radar_kafka_stack._extra_timeout }}
+    <<: *logFailedRelease
+    values:
+      - {{ .Values.radar_kafka_stack | toYaml | indent 8 | trim }}
+    set:
+      - name: kafka.replicas
+        value: {{ .Values.kafka_num_brokers }}
+      - name: kafka.replicationFactor
+        value: {{ .Values.kafka_num_topic_replicas }}
+      - name: kafka.partitions
+        value: {{ .Values.kafka_num_topic_partitions }}
+      - name: schema_registry.topic.replicationFactor
```

Comment:
Shouldn't the name be with '-'? `schema-registry.topic.replicationFactor`
Same in line 159

Reply:
Yes, indeed! I fixed this...